introduction/Infrastructure.tex

```diff
@@ -9,23 +9,42 @@ \section{Infrastructure}
 \subsection{RoboCup@Home Mailinglist}
 The official \iterm{RoboCup@Home mailing list} can be reached at
 \begin{center}
-\texttt{robocup-athome@lists.robocup.org}
+\href{mailto:robocup-athome@lists.robocup.org}{\texttt{robocup-athome@lists.robocup.org}}
 \end{center}
 You can register to the email list at:
 \begin{center}
-http://lists.robocup.org/cgi-bin/mailman/listinfo/robocup-athome
+{\small\url{http://lists.robocup.org/cgi-bin/mailman/listinfo/robocup-athome}}
 \end{center}
 
 \subsection{RoboCup@Home Web Page}
-The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at \\
+The official \iterm{RoboCup@Home website} that also hosts this RuleBook can be found at:
 \begin{center}
-http://www.robocupathome.org/
+{\small\url{https://athome.robocup.org/}}
+\end{center}
+
+\subsection{RoboCup@Home Rulebook Repository}
+The official \iterm{RoboCup@Home Rulebook Repository} is where rules are publicly discussed before applying changes, buffs, and nerfs to the rulebook.
+The entire RoboCup@Home community is welcome and encouraged to actively participate in rule discussion and crafting. The repository can be reached at:
```

Comment:
```suggestion
The entire RoboCup@Home community is welcome and encouraged to actively participate in creating and discussing the rules. The repository can be reached at:
```

I don't like the word crafting. I used **google** `define: create` and `define: crafting`

```
cre·ate
/krēˈāt/
verb
gerund or present participle: creating
bring (something) into existence.
```

```
craft·ing
/ˈkraftiNG/
noun
the activity or hobby of making decorative articles by hand.
```

Reply:
Time to go back to school and learn English again...